src/main/webapp/jsp/studentComments.jsp

```diff
@@ -1,297 +1,49 @@
-                                    id="form_commentedit-<%=commentIdx%>">
-                                    <div id="commentBar-<%=commentIdx%>">
-                                        <%
-                                        	InstructorAttributes instructor = data.roster.getInstructorForEmail(comment.giverEmail);
-                                                                                   String giverDisplay = comment.giverEmail;
-                                                                                   if(instructor != null){
-                                                                                       giverDisplay = instructor.displayedName + " " + instructor.name;
-                                                                                   }
-                                                                                   String lastEditorDisplay = null;
-                                                                                   if (comment.lastEditorEmail != null) {
-                                                                                        InstructorAttributes lastEditor = data.roster.getInstructorForEmail(comment.lastEditorEmail);
-                                                                                        lastEditorDisplay = lastEditor.displayedName + " " + lastEditor.name;
-                                                                                   }
-                                        %>
-                                        <span class="text-muted">From <b><%=giverDisplay%></b> on
-                                            <%=TimeHelper.formatDate(comment.createdAt)%> <%=comment.getEditedAtTextForStudent(giverDisplay.equals("Anonymous"), lastEditorDisplay)%>
-                                        </span>
-                                    </div>
-                                    <div id="plainCommentText<%=commentIdx%>"><%=comment.commentText.getValue()%></div>
-                                </li>
-                            </ul>
-                        </div>
-                        <%
-                        	}//comment loop ends
-                        %>
-                    </div>
-                </div>
-                <%
-                	}// check student comments ends
-                %>
-                <%
-                	int fsIndx = 0;
-                                    for (String fsName : data.feedbackResultBundles.keySet()) {//FeedbackSession loop starts
-                                        FeedbackSessionResultsBundle bundle = data.feedbackResultBundles.get(fsName);
-                                        fsIndx++;
-                %>
+</c:set>
+<ts:studentPage bodyTitle="Comments" pageTitle="TEAMMATES - Student" jsIncludes="${jsIncludes}">
+    <t:statusMessage />
+    <br>
+    <c:choose>
+    <c:when test="${not empty data.coursePagination.coursePaginationList}">
```

Comment:
While `c:choose` does resemble a `switch`, we should still indent the `c:when` block in it.